### PR TITLE
Single Bucket - Move object

### PR DESCRIPTION
**Pull request by bstopp**
## Description

Final step for single bucket and using R2 Bindings.

## How Has This Been Tested?

Added new unit and integration tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by bstopp** [comment]
All e2e tests in da-live project pass.

**Comment by bstopp** [comment]
Also, perf testing of the certain APIs:
```
> da-admin@1.0.0 test:perf
> mocha --spec=test/**/*.perf.js



  Object Performance
    ✔ copyObject handles 10000 files in folder (26836ms)
    ✔ copyObject(rename) handles 10000 files in folder (36338ms)
    ✔ move handles 10000 files in folder (36601ms)
```

File: src/helpers/copy.js [hunk 1]
```diff
@@ -10,12 +10,6 @@
  * governing permissions and limitations under the License.
  */
 
-/**
- * Creates the copy source/dest object from the current contexts
- * @param {Request} req the Request object
- * @param {Object} daCtx the DA Context
- * @return {Promise<{destination: string, source: string}|{}>}
- */
 export default async function copyHelper(req, daCtx) {
```
Comment (bosschaert) [inline src/helpers/copy.js:13]: Why is the jsdoc removed? Was it incorrect?

**Review by bosschaert — APPROVED** [review]

File: src/helpers/copy.js (hunk 1 above)
Comment (bstopp) [reply src/helpers/copy.js:19]: Didn't mean to remove it. I put it back.

**Pull request merged**